.travis.yml

```diff
@@ -8,9 +8,15 @@ osx_image:
   - xcode10.1
 compiler: clang
 
+addons:
+  homebrew:
+    update: true
+    packages:
```

Comment:
so do we no longer install cmake? Is there a system one to use?

Reply:
I mean, I guess? I didn't remove anything, and the builds passed, so praise the travis gods?

Reply:
:muscle: